src/components/dictionaryConcepts/components/ConceptNameRows.jsx

```diff
@@ -142,9 +142,10 @@ class ConceptNameRows extends Component {
             required
           >
             <option />
-            <option>Fully Specified</option>
-            <option>Synonym</option>
-            <option>Search Term</option>
+            <option value="Fully Specified">Fully Specified</option>
+            <option value="None">Synonym</option>
```

Comment:
Why does synonym have a value of None?

Reply:
This was the conclusion in the last call. Now that you mention it, I'm starting to wonder if this is right.

I've asked for clarification from Darius on talk. I think we should probably wait for that.

Reply:
Looked through the OpenMRS repo, and it seems null name_types are considered synonyms?
https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/api/ConceptNameType.java

Reply:
Which exact line did you look at?

Reply:
29

Reply:
You can click on a line number to get a full url. 😊

Reply:
https://github.com/openmrs/openmrs-core/blob/3883edf72fe0355cbdc327f36cbdd9dfd890ce06/api/src/main/java/org/openmrs/api/ConceptNameType.java#L29

Reply:
Good detective work. So what do you do with the None value, once selected?

Reply:
Yeah, no this is wrong. It should be `null`. Let me update that.
My other worry is given the way the name types are defined in that file, shouldn't ours be in all caps as well?

Reply:
Depends on what you do with the selected value.

Reply:
They are submitted as is. The api expects them like that.

Reply:
Do you know what the api does with the value of None?

Reply:
https://github.com/OpenConceptLab/oclapi/blob/d6d04d1b0c0b5e148e6c2bd8d058d09e5db9d653/ocl/nametypes_fixed.json#L1
It is a name_type of its own
Its saved it as that

Reply:
Just made the update @dkayiwa. Kindly take a look.

Reply:
Did you see the travis build failure?

Reply:
Just did. Travis barfed. Restarting the build.

Reply:
Any reason why you now have more changes than before?

Reply:
So before I had used `None` for the synonym. We've discovered it should be `null`. `null` can't be set as a value directly though, hence the conversion to and fro.